OCPP-J/src/main/java/eu/chargetime/ocpp/WebSocketListener.java

```diff
@@ -50,6 +50,8 @@ of this software and associated documentation files (the "Software"), to deal
 
   private static final int TIMEOUT_IN_MILLIS = 10000;
 
+  private static final int OCPPJ_CP_PASSWORD_LENGTH = 20;
```

Comment:
Interesting. The OCPP-J specification errata states 20 bytes, however the "Improved security for OCPP 1.6-J" document states:

"The basic authentication password is used for HTTP Basic Authentication, minimal length: 16 bytes.
It is strongly advised to be randomly generated binary to get maximal entropy. Hexadecimal represented (20 bytes maximum, represented as a string of up to 40 hexadecimal digits)."

So according to this, password lengths of 16 to 20 bytes must be accepted.

Reply:
Made a check of password length to fall into 16-20 bytes range